videostore/utils_test.go

```diff
@@ -0,0 +1,37 @@
+package videostore
+
+import (
+	"testing"
+
+	"go.viam.com/test"
+)
+
+const artifactStoragePath = "../.artifact/data/"
+
+func TestGetVideoDuration(t *testing.T) {
+	t.Run("Valid video file duration succeeds", func(t *testing.T) {
+		duration, err := getVideoDuration(artifactStoragePath + "2024-09-06_15-00-03.mp4")
+		test.That(t, err, test.ShouldBeNil)
+		test.That(t, duration.Seconds(), test.ShouldBeGreaterThan, 0.0)
```

Comment:
Will this always return a deterministic duration? If so we should test that it is what we expect, there is a `ShouldBeBetween` helper as well.

Reply:
Good call, added equal-to check for duration.